include/gdb/stacktrace.h

```diff
@@ -61,6 +62,18 @@ struct sr_gdb_stacktrace
     struct sr_gdb_frame *crash;
 
     /**
+     * TID of thread passed by Kernel to the core pattern helper as the TID
+     * of thread which triggered core dump.
+     *
+     * This value may not be set at all and is not included in GDB-produced
+     * backtrace, so it must be assigned by our callers.
+     *
+     * Its only purpose is to make the searching for the crashed thread as
+     * accurate as possible.
+     */
+    uint32_t crash_tid;
```

Comment:
Can you put the new member at the end of the struct? I think it breaks backwards binary compatibility otherwise.